### PR TITLE
Run tests against both ES v6 and ES v7

### PR DESCRIPTION
**Pull request by mgalang**
### Description

Modified circleci configuration to run the tests against Elasticsearch 6 and Elasticsearch 7

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -77,10 +35,84 @@ jobs:
...
+          - bootstrap.memory_lock=true
+          - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
+
+    steps:
+      - checkout:
+          path: ~/project/drupal/web/modules/elasticsearch_helper
+      - restore_cache:
+          keys:
+            - v2-composer-dependencies-{{ .Environment.CIRCLE_SHA1 }}
+      - start-drupal-test
```
Comment (maijs) [inline .circleci/config.yml]: Is there a way to reuse this block and change the Elasticsearch version via a parameter that is passed from the workflow jobs? 

- image: docker.elastic.co/elasticsearch/elasticsearch:<< version >>

File: .circleci/config.yml [hunk 2]
```diff
@@ -77,10 +35,84 @@ jobs:
...
+
+  test-es-6:
+    docker:
+      - image: wunderio/circleci-codeception-base
+      - image: circleci/mariadb:10.2
+        environment:
+          MYSQL_DATABASE: drupal
+          MYSQL_ROOT_PASSWORD: dbpassword
+          MYSQL_ROOT_HOST: "%"
+      - image: docker.elastic.co/elasticsearch/elasticsearch:6.8.3
```
Comment (mitrpaka) [inline .circleci/config.yml]: Latest is 6.8.10 - should we always test against latest version or fixed version?

File: .circleci/config.yml [hunk 3]
```diff
@@ -77,10 +35,84 @@ jobs:
...
+
+    steps:
+      - run:
+          name: Create Drupal
+          command: |
+            composer create-project drupal/recommended-project:8.8.6 drupal
+            cd drupal
+            COMPOSER_MEMORY_LIMIT=-1 composer require drupal/core-dev:^8
+            COMPOSER_MEMORY_LIMIT=-1 composer require drush/drush:9
+            COMPOSER_MEMORY_LIMIT=-1 composer require elasticsearch/elasticsearch:~6.0
```
Comment (mitrpaka) [inline .circleci/config.yml]: I guess we should use elasticsearch/elasticsearch:~6.0 version for es-6 testing and elasticsearch/elasticsearch:~7.0 version for es-7 testing.

File: .circleci/config.yml (hunk 1 above)
Comment (mgalang) [reply .circleci/config.yml]: Apparently not. I tried re-usable jobs yesterday but the parameter scope does not seem to reach the docker part

**Pull request merged**